src/DotLiquid.Tests/StandardFilterTests.cs

```diff
@@ -90,7 +90,6 @@ public void TestEscape()
             Assert.That(StandardFilters.Escape(null), Is.EqualTo(null));
             Assert.That(StandardFilters.Escape(""), Is.EqualTo(""));
             Assert.That(StandardFilters.Escape("<strong>"), Is.EqualTo("&lt;strong&gt;"));
-            Assert.That(StandardFilters.H("<strong>"), Is.EqualTo("&lt;strong&gt;"));
 
             Helper.AssertTemplateResult(

```

Comment:
Maybe something for another day, but I find combining `Helper.AssertTemplateResult` and direct calls to `StandardTemplate.XXX` to be a little mixed. 

The latter definitely belongs in `StandardFilterTests` as unit tests, but maybe the `Helper.Assert` tests belong somewhere else as a system level test?